components/common/typescript.gni

```diff
@@ -119,8 +125,14 @@ template("transpile_web_ui") {
       gen_output_dir = "$root_gen_dir/brave/web-ui-$resource_name"
     }
 
-    # sole output is a GRD that includes all the resources
+    # Output is a GRD that includes all the resources, and a bundle for each
+    # entry point or a GRDP file that includes all the resources.
     grd_name = "$resource_name.grd"
+    if (defined(invoker.generate_grdp) && invoker.generate_grdp) {
```

Comment:
I'm thinking not to enforce this since we can definitely have a use case where we combine assets with JS output, e.g. html page + images with JS bundle in a GRDP.